### DISCUSSION TITLE
Dynamic filtering based on current working directory

### DISCUSSION THREAD

**Discussion by ryanhanks-bestow** [invertase/melos#423] [Q&A]
Hi,

Is there a way to dynamically filter for packages based on the current working directory?

I want to be able to run a melos script from somewhere in my melos project and filter the execution to the package I'm rooted within and / or packages that are descendants if I'm within a parent package.

Apologies if I missed something obvious here.

Thanks

**Comment by blaugold** [comment]
This is not currently possible, but there is an open issue for it: https://github.com/invertase/melos/issues/311
